### PR TITLE
fix: add buttonRef prop to trigger default function

### PR DESCRIPTION
**Pull request by diffidentDude**
The buttonRef arg wasn't added to the function declaration, I have added it in

### CODE REVIEW

**Review by HeartSquared — APPROVED** [review]

**Comment by gyfchong** [comment]
`yarn changeset` please :)

File: .changeset/wise-ducks-complain.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/select": patch
+---
+
+Update tigger deftaul function to have buttonRef reference
```
Comment (gyfchong) [inline .changeset/wise-ducks-complain.md:5]:
```suggestion
Update tigger default function to have buttonRef reference
```

Comment (diffidentDude) [reply .changeset/wise-ducks-complain.md:5]: Wow, my typing really sucked!

Comment (HeartSquared) [reply .changeset/wise-ducks-complain.md]: Tigger? _* bounces *_

**Review by gyfchong — APPROVED** [review]

**Review by gyfchong — APPROVED** [review]

**Comment by gyfchong** [comment]
We are working on the yarn integrity issue 😄

**Pull request merged**